file_paths.md

```diff
@@ -163,160 +163,195 @@ The package will create the following services `puppet`, `mcollective`,
 and `pxp-agent`, all running as `LocalSystem` by default. It will not
 create a `puppet` user or group.
 
-    Path                                      Setting                        3.x
-    C:\ProgramData                                                           n/a
+    Path                                          Setting                        3.x
+    C:\ProgramData\PuppetLabs *                                                  n/a
```

Comment:
@melissa Is there a reason why we are setting this to ```C:\ProgramData\PuppetLabs``` whereas the Program Files are being installed under: ```C:\Program Files\Puppet Labs``` ?

Just asking because @McdonaldSeanp and I are running through the installed directories at the moment to make sure everything is consistent.